cmd/minikube/cmd/start.go

```diff
@@ -388,6 +423,8 @@ func runStart(cmd *cobra.Command, args []string) {
 	if err := showKubectlInfo(kubeconfig, k8sVersion, cc.Name); err != nil {
```

Comment:
I think this can be moved out of `startWithDriver` and into the main loop. It raises no error, but it also has nothing to do with drivers.